Dynavity/Dynavity/view/canvas/elements/ImageCanvasElementView.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+struct ImageCanvasElementView: View {
+    @ObservedObject var viewModel: ImageCanvasElementViewModel
+
+    init(imageCanvasElement: ImageCanvasElement) {
+        self.viewModel = ImageCanvasElementViewModel(imageCanvasElement: imageCanvasElement)
+    }
+
+    var body: some View {
+        Image(uiImage: viewModel.imageCanvasElement.image)
```

Comment:
One more thing: do we want to add a drop shadow to the image as well? Using `addCardOverlay()`

Reply:
Right, I've applied the card overlay to all canvas elements in `CanvasElementMapView`.

Reply:
Could you also remove all other instances where this `addCardOverlay()` is called? So we don't call it twice (not sure if it'll affect the appearance. Thanks!

Reply:
Done, thanks for pointing this out!